src/main/java/frc/robot/utils/MovingAverage.java

```diff
@@ -58,10 +58,10 @@ private void read(Reader reader) {
     /**
      * Retrieve an approximation for the velocity based on the distance.
      *
-     * @param distance the distance from the target.
+     * @param distance the distance from the target. [meters]
      * @return an approximation of the velocity to apply
      */
-    public double evaluateVelocity(double distance) {
+    public double evaluateVelocityByDistance(double distance) {
```

Comment:
❤️

Reply:
Approve this PR!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
Sorry, the internet uses "estimate" and "from". Should have looked it up before suggesting another name.
(For my defense, I did say "or something similar" 🙃)
```suggestion
    public double estimateVelocityFromDistance(double distance) {
```
Also, something I wrote in #5 too: if you have a function in utils, you can't use terms like "velocity" and "distance". Either move this class to the `shooter` package or use more generic terms.